src/modules/account/views/MyProfile.tsx

```diff
@@ -259,6 +291,19 @@ export const AccountMyProfile: NextPage<DefaultSeoInfo> = ({ url }) => {
 		</FormControl>
 	);
 
+	const renderLanguagePreferencesForm = () => (
+		<CheckboxList
+			items={
+				allLanguages?.map((language) => ({
+					label: tText(language.languageLabel),
```

Comment:
You can't use tText with dynamic variables, since the translation extraction script wont be able to extract them.
You'll have to make a lookup variable function:
```
const GET_TRANSLATED_LANGUAGE_LABELS = () => ({
	nl: tText('Nederlands'),
	en: tText('Engels'),
});
```

and then use it here:
```
label: GET_TRANSLATED_LANGUAGE_LABELS()[language.languageCode],
```